mhctools/process_helpers.py

```diff
@@ -108,8 +112,10 @@ def run_multiple_commands_redirect_stdout(
 
     def add_to_queue(process):
         if print_commands:
-            print(" ".join(process.args), ">",
-                  process.redirect_stdout_file.name)
+            handler = logging.FileHandler(process.redirect_stdout_file.name)
+            handler.setLevel(logging.DEBUG)
+            logger.addHandler(handler)
```

Comment:
It seems like the logger could accumulate many handlers from dead processes (sometimes this code runs tens of thousands of times), do you think that will be a problem?